doc: de-duplicate instructions in kind and hubble getting started guides (#12210)
cilium-team/cilium: Documentation/gettingstarted/kind-preload.rst

> As the ``cilium`` pod is deployed on every node, let's preload the ``cilium``
> image into the kind cluster so each worker doesn't have to pull it
> individually:

Reviewer: nit: 
```suggestion
Preload the ``cilium`` image into each worker node in the kind cluster:
```

.. parsed-literal::

  docker pull cilium/cilium:|IMAGE_TAG|

>   kind load docker-image cilium/cilium:|IMAGE_TAG|

Reviewer: This is the kind of situation where `parsed-literal` is required.
